ft/grp_config_api_test.robot

```diff
@@ -12,48 +12,64 @@ ${app_node}       app-node
 
 *** Test Cases ***
 Add group config under non-existent node, it should return failure
+  Skip If MQTT API
```

Comment:
del, ci won't run these tests with mqtt

Reply:
If you delete this, the functional test commands in the README cannot be executed smoothly